examples/terraform/vmware-cloud-director/variables.tf

```diff
@@ -77,18 +77,6 @@ variable "ssh_agent_socket" {
   type        = string
 }
 
-variable "bastion_port" {
-  description = "Bastion SSH port"
-  default     = 22
-  type        = number
-}
-
-variable "bastion_user" {
-  description = "Bastion SSH username"
-  default     = "ubuntu"
-  type        = string
-}
```

Comment:
Why don't we leave this? As far as I see, we still have the bastion VM.

Reply:
Bastion was removed completely from the setup.